march_data_collector/src/march_data_collector/CoMCalculator.py

```diff
@@ -0,0 +1,72 @@
+import rospy
+import geometry_msgs.msg
+import tf2_ros
+import tf2_geometry_msgs as tf_geo
+from urdf_parser_py.urdf import URDF
+from visualization_msgs.msg import Marker
+
+
+class CoMCalculator(object):
+
+    def __init__(self, robot, tf_listener, tf_buffer):
```

Comment:
`tf_listener` is not used, was there a reason for adding this?
```suggestion
    def __init__(self, robot, tf_buffer):
```